src/sentry/hybridcloud/options.py

```diff
@@ -161,3 +161,11 @@
     default=[],
     flags=FLAG_AUTOMATOR_MODIFIABLE,
 )
+
+# Controls whether or not the pathway to target regions via integration request contents is enabled for the system.
+register(
+    "hybrid_cloud.integration_region_targeting_rate",
+    type=Int,
```

Comment:
You might want a float here if you're using any of our rollout logic. If you remove this `type` and set the default to `0.0` options manager will figure out the type is Float.